src/cursor/cur_index.c

```diff
@@ -453,6 +453,7 @@ __wt_curindex_open(WT_SESSION_IMPL *session, const char *uri, WT_CURSOR *owner,
       __wt_cursor_notsup,                             /* remove */
       __wt_cursor_notsup,                             /* reserve */
       __wt_cursor_reconfigure_notsup,                 /* reconfigure */
+      __wt_cursor_largest_key,                        /* largest_key */
```

Comment:
Not sure index cursor should support this or not.

Reply:
I suspect it has to -- when you write the complex-table test, it will fire, I think.